sys/net/gnrc/netapi/notify/gnrc_netapi_notify.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2025 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     net_gnrc_netapi_notify
+ * @file
+ * @brief       Helpher functions to extract data from netapi notify events.
+ *
+ * @author      Elena Frank <elena.frank@tu-dresden.de>
+ * @}
+ */
+
+#include <errno.h>
+
+#include "net/gnrc/netapi/notify.h"
+#include "net/ipv6/addr.h"
+
+uint8_t gnrc_netapi_notify_copy_l2_connection_data(gnrc_netapi_notify_t *notify,
+                                                   netapi_notify_l2_connection_t *data)
+{
+    int data_len;
+
+    switch (notify->event) {
+    case NETAPI_NOTIFY_L2_CONNECTED:
+    case NETAPI_NOTIFY_L2_DISCONNECTED:
+        assert(notify->_data_len == sizeof(netapi_notify_l2_connection_t));
+
+        /* Parse event data.*/
```

Comment:
```suggestion
        /* Parse event data */
```

Just a nitpick. There should be a space between the text and the `*/`.